Modules/Quotas/src/Quotas.Domain/Aggregates/FileMetadata/FileMetadata.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Backbone.Modules.Quotas.Domain.Aggregates.FileMetadata;
+public class FileMetadata
+{
+    public int Id { get; set; }
```

Comment:
This is indeed dumb! It works because we never deal with this entity, we just count them (yes, I did test 😄). Setting this to `string` causes EF to complain about a lacking primary key. Should I define it in the Entity Configuration?

Should I apply the same fix to this?
https://github.com/nmshd/backbone/blob/6af06dee4b7889dc640ed61867a42072a0e152e1/Modules/Quotas/src/Quotas.Domain/Aggregates/Messages/Message.cs#L4

Reply:
> Setting this to string causes EF to complain about a lacking primary key. Should I define it in the Entity Configuration?

Really? That's weird.  If there's no other possibility, you have to explicitly configure it as primary key, yes.

> Should I apply the same fix to this?
Yes please.